stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/CallState.kt

```diff
@@ -306,6 +316,22 @@ public class CallState(
     private val _settings: MutableStateFlow<CallSettingsResponse?> = MutableStateFlow(null)
     public val settings: StateFlow<CallSettingsResponse?> = _settings
 
+    private val _durationInMs = flow {
+        while (scope.isActive) {
```

Comment:
I think we could use `currentCoroutineContext().isActive` here - this way we don't need to specify  `CoroutineScope`. I did some quick tests and it seems to behave the same.

Reply:
Sounds better!